src/migrator.ts

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import * as os from 'os';
+import * as path from 'path';
+import * as Sequelize from 'sequelize';
+import * as Umzug from 'umzug';
+
+import {SequelizeConnectionManager} from '@essential-projects/sequelize_connection_manager';
+
+const sequelizeConnectionManager: SequelizeConnectionManager = new SequelizeConnectionManager();
+
+// Based on: https://github.com/abelnation/sequelize-migration-hello/blob/master/migrate.js
+export async function migrate(env: string, database: string, sqlitePath: string): Promise<void> {
+
+  sqlitePath = getFullSqliteStoragePath(sqlitePath);
+
+  const sequelizeInstanceConfig: any = env === 'sqlite'
+    ? createSqLiteConfig(sqlitePath, database)
+    : createPostgresConfig(database);
+
+  const sequelizeInstance: Sequelize.Sequelize = await sequelizeConnectionManager.getConnection(sequelizeInstanceConfig);
+
+  const umzugInstance: Umzug.Umzug = await createUmzugInstance(sequelizeInstance, database);
+  await umzugInstance.up();
+
+  await sequelizeConnectionManager.destroyConnection(sequelizeInstanceConfig);
+}
+
+function createSqLiteConfig(sqlitePath: string, store: string): any {
```

Comment:
also here maybe 
```suggestion
function createSqLiteConfig(sqlitePath: string, store: string): object {
```

but idk

Reply:
I used `any` here instead of `Sequelize.Options`, because there is currently an issue in the Sequelize typings, which causes the option `supportBigNumbers` to not be recognized, despite it being a perfectly valid parameter to use.
So if I were to use `Sequelize.Options` here, it would cause a tslint error.

But I guess `object` should work just as well.